keps/sig-node/2570-memory-qos/README.md

```diff
@@ -580,6 +587,10 @@ feature flags will be enabled on some API servers and not others during the
 rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
+Already running workloads will not have `memory.min/memory.high` set at Pod level. Only `memory.min` will be
+set at Node level cgroup when the kubelet restarts. The existing workloads will be impacted only when kernel
+isn't able to maintain at least `memory.min` level of memory for the non-guaranteed workloads within the
+Node level cgroup.
 
 ###### What specific metrics should inform a rollback?
 
```

Comment:
There is not, I don't think we wanted to expose memory.high as it's a bit of implementation detail (i.e. it is a function of the throttling factor set).

However, long term, it may make sense to expose (maybe via cadvisor) some of the memory events, e.g, `memory.high` field of `memory.events` which can inform how many times `memory.high` was breached and the cgroup was throttled. [1]

https://docs.kernel.org/admin-guide/cgroup-v2.html

Reply:
I thnk the metric for how many times it was breached needs to be in beta, not GA. That seems like a very useful thing for operators to know if they start seeing issues.

Also describe the types of issues users may see with their workloads. When under memory pressure from breaching the "high", will they see failures? Slow downs?

Reply:
Makes sense, I chatted with @ndixita, and I agree we should have some type of metrics that folks can observe to see this. We added this to the beta criteria.